Label_Microservice/README.md

```diff
@@ -0,0 +1,73 @@
+# Repo-Specific Label Microservice
```

Comment:
Thanks @abcdefgs0324 . Could you please add the following information

1. The document you wrote for onboarding new repositories; can you port that over to a markdown doc and link to it from this doc?
1. Can you provide instructions for how you update the staging/production deployments?
    * e.g. how do you build the docker images?
    * which Kubernetes YAML files need to be updated?
    * How you apply those changes?

1. Can you provide instructions for how you control how much traffic is directed to the new model?